settings.gradle

```
@@ -12,8 +12,9 @@ includeWithFlags ':client:java-armeria-legacy',             'java', 'publish', '
 includeWithFlags ':client:java-armeria-xds',                'java', 'publish', 'relocate'
 includeWithFlags ':client:java-spring-boot2-autoconfigure', 'java', 'publish', 'relocate', 'no_aggregation'
 includeWithFlags ':client:java-spring-boot2-starter',       'java', 'publish', 'relocate', 'no_aggregation'
-includeWithFlags ':client:java-spring-boot3-autoconfigure', 'java17', 'publish', 'relocate'
-includeWithFlags ':client:java-spring-boot3-starter',       'java17', 'publish', 'relocate'
+includeWithFlags ':client:java-spring-boot3-autoconfigure', 'java17', 'spring-boot3', 'publish', 'relocate'
+includeWithFlags ':client:java-spring-boot3-starter',       'java17', 'spring-boot3', 'publish', 'relocate'
+includeWithFlags ':cloud-native:boot-loader',               'kotlin', 'spring-boot3'
```

Question) Have you tried specifying `java17`? I think it could help avoid manually setting the supported minimum java version in gradle scripts

```suggestion
includeWithFlags ':cloud-native:boot-loader',               'java17', 'kotlin', 'spring-boot3'
```